rsa-cryptosystem-signature/index18.js

```diff
@@ -0,0 +1,34 @@
+const firstPrime = 2;
+const secondPrime = 5;
+const N = firstPrime * secondPrime;
+const phiOfN = 0;
+let publicKey = 0;
+
+function hashTheMessage(message) {
+  let hashValue = 0;
+  for (let i = 0, msgLength = message.length; i < msgLength; ++i) {
+    hashValue += message.charCodeAt(i);
+  }
+  return hashValue;
+}
+
+/*
+Continuing with our example of 12 and 9. We observe, 3 divide both the numbers evenly. Thus, they are not coprime.
```

Comment:
Going back to our example numbers 12 and 9, both can be divided by 3 evenly.
Thus, they are not co-prime.
Return `false` if `smallerNum` and `largerNum` can be divided by `i` evenly.